includes/general.php

```diff
@@ -2346,6 +2346,106 @@ function pods_no_conflict_off( $object_type = 'post' ) {
 	return false;
 }
 
+/**
+ * Returns a list of all WordPress and Pods reserved keywords.
+ *
+ * @link https://codex.wordpress.org/Reserved_Terms
+ *
+ * @since 2.7.15
+ * @return array
+ */
+function pods_reserved_keywords() {
+
+	$keywords = array(
+		'attachment',
+		'attachment_id',
+		'author',
+		'author_name',
+		'calendar',
+		'cat',
+		'category',
+		'category__and',
+		'category__in',
+		'category__not_in',
+		'category_name',
+		'comments_per_page',
+		'comments_popup',
+		'custom',
+		'customize_messenger_channel',
+		'customized',
+		'cpage',
+		'day',
+		'debug',
+		'embed',
+		'error',
+		'exact',
+		'feed',
+		'hour',
+		'link_category',
+		'm',
+		'minute',
+		'monthnum',
+		'more',
+		'name',
+		'nav_menu',
+		'nonce',
+		'nopaging',
+		'offset',
+		'order',
+		'orderby',
+		'p',
+		'page',
+		'page_id',
+		'paged',
+		'pagename',
+		'pb',
+		'perm',
+		'post',
+		'post__in',
+		'post__not_in',
+		'post_format',
+		'post_mime_type',
+		'post_status',
+		'post_tag',
+		'post_type',
+		'posts',
+		'posts_per_archive_page',
+		'posts_per_page',
+		'preview',
+		'robots',
+		's',
+		'search',
+		'second',
+		'sentence',
+		'showposts',
+		'static',
+		'subpost',
+		'subpost_id',
+		'tag',
+		'tag__and',
+		'tag__in',
+		'tag__not_in',
+		'tag_id',
+		'tag_slug__and',
+		'tag_slug__in',
+		'taxonomy',
+		'tb',
+		'term',
+		'terms',
+		'theme',
+		'title',
+		'type',
+		'w',
+		'withcomments',
+		'withoutcomments',
+		'year',
+	);
+
+	// @todo: Pods reserved keywords.
+
```

Comment:
```suggestion
	/**
	 * Filter the WordPress and Pods reserved keywords.
	 *
	 * @since 2.7.15
	 *
	 * @param array $keywords List of WordPress and Pods reserved keywords.
	 */
```